DependencyInjection/Configuration.php

```diff
@@ -34,6 +34,14 @@ public function getConfigTreeBuilder()
                     ->prototype('scalar')
                     ->end()
                 ->end()
+                ->arrayNode('extractor')
+                    ->prototype('array')
+                        ->children()
+                            ->scalarNode('extensions')->defaultValue(array('js', 'jsx', 'ts'))->end()
```

Comment:
this is not a scalar node. This is a prototyped array node. Otherwise, users won't be able to configure multiple values.